src/components/sidebar/Sidebar.tsx

```diff
@@ -48,7 +48,7 @@ const pagesProperties: PageProperty[] = [
     name: MainPages.dashboard,
     marginBottom: "mb-4",
     icon: <RectangleGroupIcon className="h-[1.125rem]" />,
-    link: routePaths.dashboardPage(),
+    link: "/dashboard",
```

Comment:
why did you change this?

Reply:
> why did you change this?

Okay, I fixed the actual source of the problem. In the route paths, the dashboard page was pointing to '/' instead of '/dashboard'.